### PR TITLE
Dzollo/star 748/position messages

### PR DESCRIPTION
**Pull request by denniszollo**

### CODE REVIEW

File: spec/yaml/swiftnav/sbp/navigation.yaml [hunk 1]
```diff
@@ -825,6 +825,499 @@ definitions:
...
+        - flags:
+            type: u8
+            desc: Status flags
+            fields:
+              - 5-7:
+                  desc: Reserved
+              - 3-4:
+                  desc: Inertial Navigation Mode
+                  values:
+                      - 0: None
```
Comment (switanis) [inline spec/yaml/swiftnav/sbp/navigation.yaml]: Need to make them reserved

File: spec/yaml/swiftnav/sbp/navigation.yaml [hunk 2]
```diff
@@ -825,6 +825,499 @@ definitions:
...
+                      - 1: INS used
+              - 0-2:
+                  desc: Fix mode
+                  values:
+                      - 0: Invalid
+                      - 1: Single Point Position (SPP)
+                      - 2: Differential GNSS (DGNSS)
+                      - 3: Float RTK
+                      - 4: Fixed RTK
+                      - 5: Dead Reckoning
```
Comment (switanis) [inline spec/yaml/swiftnav/sbp/navigation.yaml]: Make as reserved

**Comment by switanis** [comment]
Looks good, except one message where flags are not made reserved. See comments above.
Thanks for doing this!

File: spec/yaml/swiftnav/sbp/navigation.yaml (hunk 1 above)
Comment (denniszollo) [reply spec/yaml/swiftnav/sbp/navigation.yaml]: done thanks for finding that!

**Pull request merged**